webapp/src/components/HOC/withAuthorizedAction/AuthorizationModal/AuthorizationModal.tsx

```diff
@@ -73,7 +77,16 @@ export function AuthorizationModal({
         action: t('mana_authorization_modal.confirm_transaction.action'),
         onActionClicked: handleAuthorized,
         testId: 'confirm-action-step',
-        status: AuthorizationStepStatus.PENDING
+        status: confirmationStatus,
+        isLoading:
```

Comment:
using the previous declaration

```suggestion
        isLoading,
```